arc-lang/examples/strings.arc

```diff
@@ -0,0 +1,31 @@
+# RUN: arc-lang %s | arc-mlir-rust-test %t - -rustinclude %s.rust-tests
+# RUN: arc-lang %s | arc-mlir-rust-test %t-canon - -rustinclude %s.rust-tests -canonicalize
```

Comment:
Use
```
# RUN: arc -o %t run %s -- -rustinclude %s.rust-tests
# RUN: arc -o %t-canon run %s -- -rustinclude %s.rust-tests -canonicalize
```

And you'll avoid all the crashes from `arc-lang` not finding its runtime libraries.